Sources/TuistKit/Services/GraphService.swift

```diff
@@ -67,6 +76,8 @@ final class GraphService {
             try exportDOTRepresentation(from: graph, at: filePath)
         case .png:
             try exportPNGRepresentation(from: graph, at: filePath, layoutAlgorithm: layoutAlgorithm)
```

Comment:
Not part of this PR, but if the LayoutAlgorithm is only relevant for png, why isn't it an associated value of it?

Reply:
I'm going to resolve this here since as you mentioned, this is not relevant to this PR.